### ISSUE TITLE
Accessibility - <html> element missing `lang` attribute

### ISSUE THREAD

**Issue by vkoves** [chihacknight/decarbonize-my-state#81]
From [AxeDevTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd):

![Screenshot from 2022-05-10 21-24-49](https://user-images.githubusercontent.com/3187531/167756421-aad96e4b-7f56-4078-b34d-2cd6b6da47dd.png)

In HTML this is as simple as adding `lang="en"` to the `<html>` element, but Gatsby doesn't let us easily access this (😭) so we likely need to add a package for this 🙃

**Comment by vkoves** [comment]
@nofurtherinformation said we should add [the Helment package](https://www.npmjs.com/package/helmet) which lets us manipulate `<head>` data in Gatsby.
